### DISCUSSION TITLE
split repository for Addon Home Assistant ?

### DISCUSSION THREAD

**Discussion by chpego** [wmbusmeters/wmbusmeters#788] [Ideas]
it might be easier to separate from the current repository, the plugin for Home Assistant in another repository?

**Comment by BIBOLV** [comment]
Done in https://github.com/wmbusmeters/wmbusmeters-ha-addon 😉

**Reply by chpego** [reply]
When i try to install from new repository (not edge version):
![image](https://user-images.githubusercontent.com/38792705/224514553-f57a76c1-d3b4-42f5-a8fc-893298932f6c.png)

Maybe a problem here https://github.com/wmbusmeters/wmbusmeters-ha-addon/blob/530500247ae723d719df9a813b0ecfd61d38e2ac/wmbusmeters-ha-addon/config.json#L8

**Reply by BIBOLV** [reply]
Stable version is not been built yet, currently only edge version is working.

**Reply by BIBOLV** [reply]
Stable version also is built now, FYI.
